src/mlservice.h

```diff
@@ -270,6 +271,24 @@ namespace dd
             }
         }
 
+      // labels
+      if (labels)
+        {
+          auto labels_vec = oatpp::Vector<oatpp::String>::createShared();
+
+          if (!this->_mlmodel._hcorresp.empty())
+            {
+              labels_vec->reserve(this->_mlmodel._hcorresp.size());
+
+              for (const auto &kv : this->_mlmodel._hcorresp)
+                {
+                  labels_vec->push_back(kv.second);
+                }
+            }
+          // TODO exception if no labels
```

Comment:
Is this exception from oatpp?

Reply:
That's just a note I forgot to remove :) Was thinking of triggering an exception if the model is not supposed to have labels, but hcorresp can also be empty if the model has no corresp file, so triggering an exception would be confusing.

Reply:
I'm removing the TODO, thanks for catching that up